x-pack/test_serverless/functional/test_suites/search/navigation.ts

```diff
@@ -71,11 +72,11 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
     });
 
     it('navigate to playground from side nav', async () => {
+      await header.waitUntilLoadingHasFinished();
       await svlCommonNavigation.sidenav.clickLink({ deepLinkId: 'searchPlayground' });
```

Comment:
I think these lines should actually be swapped: waiting fore loading to finish after navigation to the playground UI and before starting the assertions.